decidim_app-design/app/assets/stylesheets/features/banners/_block-banner.scss

```diff
@@ -0,0 +1,34 @@
+.block-banner{
```

Comment:
This isn't scoped to consultations - is it meant to be used in other general cases? If so, we should probably promote this to the core (styles are found in `decidim-core/app/assets/stylesheets`).

Reply:
@josepjaume Yes, it could probably be as a different feature. Sorry about the Sass structure, I wasn't sure about the right way of structuring feature/plugin and core styles. Could you please reorganise it?